Source/WebGPU/WGSL/ConstantValue.h

```diff
@@ -150,6 +166,21 @@ struct ConstantValue : BaseValue {
             return *abstractInt;
         RELEASE_ASSERT_NOT_REACHED();
     }
+    half getHalf() const
```

Comment:
Same here, we shouldn't be implicitly converting from integral values to floats, as that can only be done via explicit operations according to the language semantics.

Reply:
oh, do you mean WGSL semantics?  Sure, I can remove the integer cases.

Reply:
I've removed the integer and bool cases.